core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -39,7 +39,7 @@ public BoundedOffHeapDataContainer(int desiredSize, long maxSize, EvictionType t
       if (type == EvictionType.COUNT) {
          sizeCalculator = i -> 1;
       } else {
-         // Use size of entry plus 28 for our LRU pointer node
+         // Use size of entry plus 16 for our LRU pointers
          sizeCalculator = i -> offHeapEntryFactory.getSize(i) + OffHeapLruNode.getSize();
```

Comment:
Doesn't `offHeapEntryFactory.getSize()` include the size of the LRU pointers?

Reply:
Ah yes it very well might now that I consolidated them into a single block.